### ISSUE TITLE
OCADU a11y audit request

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#1293]
Related Slack convo: https://gcdigital.slack.com/archives/C011SV2PM1B/p1690292078421249
Card from last year: https://app.zenhub.com/workspaces/notify-planning-614b3ad91bc2030015ed22f5/issues/gh/cds-snc/notification-planning/447

Going to ask for review from Phil and Yael

**Comment by adriannelee** [comment]
Drafted the request but I don't have edit access to the doc.

**Comment by yaelberger-commits** [comment]
Mohamed sent off to Jesse B. Done!
